setup.py

```diff
@@ -60,6 +60,7 @@ def required(requirements_file):
     license='Apache-2.0',
     url='https://github.com/OpenVoiceOS/ovos-core',
     description='mycroft-core packaged as a library',
+    obsoletes=["mycroft-core"],
```

Comment:
reverted, expects package name not pypi name, so cant contain - and "mycroft" is also the package for ovos-core, that was the point of adding this here in first place, since we cant add "mycroft-core" i will drop this

https://stackoverflow.com/questions/16127080/setup-py-expected-parenthesized-list-when-required-package-name-contains-a-dash